<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Snake 2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>960</width>
     <height>540</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>960</width>
     <height>540</height>
    </size>
   </property>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>5</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>32</y>
      <width>950</width>
      <height>503</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>950</width>
      <height>503</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>950</width>
      <height>503</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="winOrLoseText">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>30</y>
      <width>949</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>5</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>5</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberSec">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>5</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="timeText">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>5</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberScore">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>5</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="scoreText">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>5</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Score</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberHighscore">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>5</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="HighscoreText">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>5</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Highscore</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="levelSelecter">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>5</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="levelText">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>5</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Level</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>5</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>654</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
